lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/TransactionSpec.hs

```diff
@@ -118,6 +113,18 @@ mkStdTxSpec = do
     let block0 = Hash $ unsafeFromHex
             "13c3d835c53a198f7c8513b04d99eeb23c745c0a73364c2f0e802fa38eec9dba"
 
+    -- Bash equivalent of above:
+    -- XPRV0HEX=30c8422fd3cbaf54449df9f627a1f88cf85d4fee84083a91cdb6f0dbdb09c24ded29409c6a8275a643a2e79d280d97a898a3366706f3f677434b75c9d98680d0
+    -- XPRV1HEX=b81e217576bf2683f3359f50d0bf938ca3c61fdf7a2d0c1b2b35f7fb174dc0426bb7377c9ea9cb481b4c3df601379fbf69033add18c1d272d7975c43682afc48
+    -- XPRV2HEX=e01bfb39e3e595fce0b9b19e386a82816e0cef8aa823c75a32bc0f39bcf7c14e8a03d255df0440b6d0fcf5d5199a582d1df7d858bd7556d4941ebf6223fa66d1
+    -- XPRV0=$(echo $XPRV0HEX | jcli key from-bytes --type ed25519Extended)
+    -- XPRV1=$(echo $XPRV1HEX | jcli key from-bytes --type ed25519Extended)
+    -- XPRV2=$(echo $XPRV2HEX | jcli key from-bytes --type ed25519Extended)
```

Comment:
We _could_ have a bash script for generating all the goldens to files, and then `BS.readFile` from this module.